settings.py

```diff
@@ -20,6 +20,10 @@
 VELLUM_DEBUG = None
 
 
+# For loading es6 requirejs modules in a local environment, set this to True
+IS_LOCAL_REQUIREJS_ES6 = DEBUG
```

Comment:
Should this be below where `localsettings` is imported to allow `DEBUG` overrides to apply to this variable?

Aside: I'm surprised `DEBUG` defaults to true.